### ISSUE TITLE
tf.keras.layers.Normalization(invert=True) does not restore from save

### ISSUE THREAD

**Issue by mzakharo** [keras-team/keras#17556]
**System information**.
- Have I written custom code: yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 22.04
- TensorFlow installed from (source or binary):  tf-nightly from pip
- TensorFlow version (use command below): 2.13.0-dev20230212
- Python version:  3.10.6
- Bazel version (if compiling from source):
- GPU model and memory:  CPU/N/A
- Exact command to reproduce: run the code below:

**Describe the problem**.

`keras.models.load_model()` does not load `tf.keras.layers.Normalization` layer with `invert=True`.

**Describe the current behavior**.

It seems to omit `invert=True` flag when loading (or saving) the model.

**Describe the expected behavior**.

original trained and saved/loaded model output the same when using `Normalization` with `invert=True`.

**Standalone code to reproduce the issue**.

```python
import numpy as np
import pandas as pd
import sys
import tensorflow as tf
from tensorflow import keras
from tensorflow.keras import layers
print(tf.__version__)

dataset = pd.read_csv('https://raw.githubusercontent.com/mzakharo/micropython/tubby/ports/esp32/orp2.csv')
train_dataset = dataset.sample(frac=0.9, random_state=0)
train_features = train_dataset.copy()
train_labels = train_features.pop('orp')

inorm = tf.keras.layers.Normalization(axis=-1, input_shape=[2, ])
inorm.adapt(np.array(train_features))
onorm = tf.keras.layers.Normalization(axis=-1, invert=True)
onorm.adapt(train_labels)

def build_and_compile_model(inorm, onorm):
  model = keras.Sequential([
      inorm,
      layers.Dense(10, activation='relu'),
      layers.Dense(1), 
     onorm,
  ])
  model.compile(loss='mean_squared_error', optimizer='adam')
  return model
model = build_and_compile_model(inorm, onorm)
history = model.fit(
    train_features,
    train_labels,
    validation_split=0.1,
    verbose=0, epochs=10)
fname = 'model.h5'
model.save(fname)

p0 = 7.6 
p1 = 0.85
v = model.predict([(p0, p1)], verbose=0)[0]
print('original output: ', v)
lmodel = keras.models.load_model(fname)
v = lmodel.predict([(p0, p1)], verbose=0)[0]
print('restored output: ', v)


```

**logs**.
```
original output:  [667.36383]
restored output:  [-8.889455]
```

**Comment by Frightera** [comment]
I was able to reproduce the bug, please find [the gist here](https://colab.research.google.com/gist/Frightera/0323a8686352fa949a77fb21d2f6f125/17556-repro.ipynb). Thanks!

@mzakharo Do you want to create a PR for that or I can investigate and proceed with the fix?

**Comment by tilakrayal** [comment]
@mzakharo,
The PR https://github.com/keras-team/keras/pull/17501 which was raised for the similar issue was merged. The PR made   the **invert** attribute is tracked when saving and loading. Also I tried to execute the mentioned code and the output was also as expected. 
**Output Before:**

```
original output:  [667.36383]
restored output:  [-8.889455]
```

**Output After:**

```
2.13.0-dev20230215
original output:  [655.1863]
restored output:  [655.1863]
```

Kindly find the gist of it [here](https://colab.research.google.com/gist/tilakrayal/341e1c93b772c1c1af3213e4bbbd2422/untitled951.ipynb). Thank you!

**Comment by mzakharo** [comment]
Thank you

**Comment by tilakrayal** [comment]
@mzakharo,
Glad the issue is resolved for you, please feel free to move this to closed status. Thank you!
